### PR TITLE
Simplify Casting with ReactiveJwtDecoders

### PR DESCRIPTION
**Pull request by ngocnhan-tran1996**
Issue: https://github.com/spring-projects/spring-security/issues/15773

Please tell me if I miss something

### CODE REVIEW

File: oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/ReactiveJwtDecoders.java [hunk 1]
```diff
@@ -49,11 +49,12 @@ private ReactiveJwtDecoders() {
 	 * @return a {@link ReactiveJwtDecoder} that was initialized by the OpenID Provider
 	 * Configuration.
 	 */
-	public static ReactiveJwtDecoder fromOidcIssuerLocation(String oidcIssuerLocation) {
+	@SuppressWarnings("unchecked")
```
Comment (jzheaux) [inline oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/ReactiveJwtDecoders.java:52]: Will you please update the copyright year to `2024` at the top of the file?

**Review by jzheaux — CHANGES_REQUESTED** [review]
Thanks for the PR, @ngocnhan-tran1996! Will you please ensure that there is only one commit in your PR and that it's description is in the following format?

```
Provide Casting for ReactiveJwtDecoder

Closes gh-15773
```

File: oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/ReactiveJwtDecoders.java (hunk 1 above)
Comment (ngocnhan-tran1996) [reply oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/ReactiveJwtDecoders.java:52]: I changed commit, thanks for your review

**Review by jzheaux — APPROVED** [review]

**Comment by jzheaux** [comment]
Thanks, @ngocnhan-tran1996! This is now merged into `main`.

**Pull request merged**